webapp/src/main/webapp/src/assets/i18n/en.json

```diff
@@ -30,16 +30,19 @@
       }
     },
     "groups": {
-      "title": "Groups",
+      "title": "Student Group Administration",
       "select-school": "School",
       "select-school-year": "School Year",
       "select-subject": "Subject",
-      "no-results": "No groups found",
+      "no-results": "No groups found matching the given filter criteria.",
+      "no-schools": "Your user account does not have access to view the groups of any schools.",
```

Comment:
Maybe we can add the "please see your administrator if you feel your access is incorrect" message?
If it's a page view we might just want to throw up a 404 page no?